src/main/java/Controller.java

```diff
@@ -1,41 +1,153 @@
+
+        // create menuBar for game scene (welcome Menu + newLooks option)
+        gameSceneMenu = this.createMenuBar().getMenus().get(0);
+        gameSceneMenu.getItems().add(newLookMenuItem);
+        gameSceneMenuBar = new MenuBar(gameSceneMenu);
+        gameScene.getChildren().add(gameSceneMenuBar);
+
+        primaryStage.setTitle("Keno Game");
+        primaryStage.setScene(new Scene(welcomeScene, Util.width, Util.height));
+        primaryStage.show();
+    }
+
+
+    public void handle(Event event) {
+
+        if (event.getSource() == startGameBtn){
+            primaryStage.setScene(new Scene(gameScene,Util.width,Util.height));
+
+        }
+        else if (((MenuItem)event.getSource()).getId() == "rulesBtn"){
+            displayGameInfo("Rules for the game:",Util.gameRules );
+        }
+        else if (((MenuItem)event.getSource()).getId() == "oddsBtn"){
+            displayGameInfo("Odds of Winning the Game:",Util.oddsOfWinning);
+        }
+        else if (((MenuItem)event.getSource()).getId() == "newLookMenu"){    //event handling for new Looks menu item in the game scene menu
+            countLookChanges++;
+            if(countLookChanges%2 ==0){ //switch back to old look
+                setNewLooks(false);
+            }
+            else{            // set a new look
+                setNewLooks(true);
+            }
+        }
+
+        else {
+            primaryStage.close();
+        }
+
+    }
+
+    private MenuBar createMenuBar(){
         // create menu items
         MenuItem rulesMenuItem = new MenuItem("Display the rules of the game");
         MenuItem oddsMenuItem = new MenuItem("Display the odds of winning");
         MenuItem exitMenuItem = new MenuItem("Exit game");
+
+        rulesMenuItem.setId("rulesBtn");
+        oddsMenuItem.setId("oddsBtn");
+        exitMenuItem.setId("exitBtn");
+
+
+        rulesMenuItem.setOnAction(this);
+        oddsMenuItem.setOnAction(this);
+        exitMenuItem.setOnAction(this);
+
+        Menu welcomeMenu = new Menu("Menu");
         // add the menu items to menu
-        menu.getItems().addAll(rulesMenuItem, oddsMenuItem, exitMenuItem);
+        welcomeMenu.getItems().addAll(rulesMenuItem, oddsMenuItem, exitMenuItem);
 
+        MenuBar menuBar = new MenuBar();
+        menuBar.getMenus().add(welcomeMenu);
 
+        return menuBar;
+        // commit
 
+    }
 
-        root.getChildren().add(menuBar);
+    //Display the Rules and Odds of winning using an alert
+    public void displayGameInfo(String message, String menuInfo){
+
+        Alert alert = new Alert(Alert.AlertType.NONE);
+        alert.setTitle(message);
+        alert.setContentText(menuInfo);
+        alert.show();
+        alert.getDialogPane().getButtonTypes().add(ButtonType.OK);
 
-        primaryStage.setTitle("Keno Game");
-        primaryStage.setScene(new Scene(root, 750, 450));
-        primaryStage.show();
     }
-}
+
+// change the looks of the game scene
```

Comment:
- Can we have the plain default look and another look too? So in total, still 2 looks
- The green background comes in the way of the green in the grid